src/solver/optim/api/include/antares/optim/api/LinearProblemData.h

```diff
@@ -27,45 +27,74 @@
 #pragma once
 
 #include <utility>
-#include "antares/solver/simulation/sim_structure_probleme_economique.h"
+#include <vector>
 
-#include "vector"
+#include "antares/solver/simulation/sim_structure_probleme_economique.h"
 
 namespace Antares::optim::api
 {
-    class LinearProblemData final
+class LinearProblemData final
+{
+public:
+    using ScalarDataDict = std::map<std::string, std::vector<double>>;
+
+    using TimedData = std::vector<double>;
+    using TimedDataDict = std::map<std::string, std::vector<TimedData>>;
```

Comment:
If I understand well, this implicitely means that 1st dimension is scenario and 2nd dimension is time.
I find it hard to read & maintain in time and over different files. Is there a way to have a multi-key map instead? Or define an extra struct / class? Or at least explicit it in the name of the attribute?

Reply:
Sure !